js/components/testimonialsList/testimonialsList.js

```diff
@@ -0,0 +1,31 @@
+function testimonialsList(selector, data) {
+  const DOM = document.querySelector(selector);
+  let html = '';
+  for (const item of data) {
+    html += `<div class="col-12 testimonials-slider-content">
+      <div class="testimonials-card">
+        <div class="testimonials-content">
+          <img
+            src="./img/blog-${item.picture}-img.webp"
+            alt="author"
+            class="testimonials-img"
+          />
+          <p class="section-description">
+          ${item.text}
+          </p>
+          <h3 class="section-smaller">${item.feedbackProvider}</h3>
+          <p class="section-description">${item.job}</p>
+        </div>
+      </div>
+    </div>
+    <div class="testimonials-actions">
+      <span class="lnr lnr-arrow-up carousel-btn"></span>
+      <span class="lnr lnr-arrow-down carousel-btn"></span>
+    </div>`;
```

Comment:
kad rodykles nesidubliuotu šita kodą reiktų iškelti iš for sąlygos. Ir tiesiog prijungti prie html sukurto stringo po for'o. Tikiuosi aiškiai parašiau :)

Reply:
labai aiskiai :) aciu!